src/io/mod.rs

```diff
@@ -305,13 +305,9 @@ cfg_std! {
 }
 
 cfg_default! {
-    // For use in the print macros.
-    #[doc(hidden)]
-    pub use stdio::{_eprint, _print};
-
-    pub use stderr::{stderr, Stderr};
-    pub use stdin::{stdin, Stdin};
-    pub use stdout::{stdout, Stdout};
+    pub use stderr::{stderr, Stderr, StderrLock};
+    pub use stdin::{stdin, Stdin, StdinLock};
+    pub use stdout::{stdout, Stdout, StdoutLock};
```

Comment:
We should probably remove these; they're not very useful at the moment.

```suggestion
```

Reply:
Deleted them